content/events/2018/06/2018-06-01-us-web-design-system-monthly-call.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: us-web-design-system-monthly-call
+title: 'U.S. Web Design System Monthly Call'
+summary: 'Join us for our monthly conversation with designers and developers across the government.'
+featured_image:
+  uid:
+  alt: ''
+event_type:
+  - online
+  - zoom
+date: 2018-06-01 13:00:00 -0500
+end_date: 2018-06-01 13:30:00 -0500
+event_organizer: DigitalGov University
+host: U.S. Web Design System
+registration_url: https://www.eventbrite.com/e/april-us-web-design-system-call-tickets-44852101844
+
+---
+
+Join us on Friday, June 1, at 10 am Pacific / 1 pm Eastern, for our monthly conversation with fellow designers and developers across the government.
+
+This month, [the U.S. Web Design](https://designsystem.digital.gov/) team will talk with Jacqueline Gu about how her team has been using the USWDS on the redesign of the Drug Enforcement Agency (DEA) site. She’ll also share the  design system which they’ve expanded upon.
```

Comment:
Please change:
U.S. Web Design team --> U.S. Web Design System team

[the U.S. Web Design](https://designsystem.digital.gov/) --> the [U.S. Web Design System](https://designsystem.digital.gov/)

Reply:
Fixed.